packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -244,6 +246,8 @@ jobs:
     with:
       package-name: ${package.name}
       working-directory: $repoRelativePath
+      ${hasGoldens ? 'runner-os: macos-latest' : ''}
+      ${hasGoldens ? 'has-goldens: true' : ''}
```

Comment:
This creates empty lines in other generated workflows when running the generate command. 

Couldn't find a simple way to trim the extra lines when not needed...

@dnys1 would it be better to have them always defined?